dump_utils.cpp

```diff
@@ -51,23 +51,45 @@ std::string getService(sdbusplus::bus::bus& bus, const std::string& path,
 
 BootProgress getBootProgress()
 {
+    BootProgress bootProgessStage;
     constexpr auto bootProgressInterface =
         "xyz.openbmc_project.State.Boot.Progress";
     // TODO Need to change host instance if multiple instead "0"
     constexpr auto hostStateObjPath = "/xyz/openbmc_project/state/host0";
+    std::string value =
+        getStateValue(bootProgressInterface, hostStateObjPath, "BootProgress");
+    bootProgessStage = sdbusplus::xyz::openbmc_project::State::Boot::server::
+        Progress::convertProgressStagesFromString(value);
+    return bootProgessStage;
+}
 
-    BootProgress bootProgessStage;
+HostState getHostState()
```

Comment:
except isHostQuiesced we can put other newly added methods in internal namespace as they are helper functions or define them only in the cpp file

Reply:
Those can be used independently too